services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/EnhancedClientUtils.java

```diff
@@ -116,8 +116,9 @@ public static String cleanAttributeName(String key) {
         AttributeValue partitionKeyValue = tableSchema.attributeValue(item, partitionKeyName);
         Optional<AttributeValue> sortKeyValue = sortKeyName.map(key -> tableSchema.attributeValue(item, key));
 
-        return sortKeyValue.map(attributeValue -> Key.create(partitionKeyValue, attributeValue))
-                           .orElseGet(() -> Key.create(partitionKeyValue));
+        return sortKeyValue.map(
+            attributeValue -> Key.builder().partitionValue(partitionKeyValue).sortValue(attributeValue).build())
```

Comment:
Can you newline partitionValue, sortValue and build so that this can then be formatted nicer within our line length limits?